### PR TITLE
Update Async API Spec

### PR DESCRIPTION
**Pull request by johnwatson484**
https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/164440

### CODE REVIEW

File: docs/asyncapi.yaml [hunk 1]
```diff
@@ -31,356 +31,444 @@ channels:
...
-                  description: Unique scheme code for funding option
-                fundCode:
-                  type: string
-                  description: Unique fund code to pay from
-                accountCode:
-                  type: string
-                  description: Unique account code to pay from
+        $ref: '#/components/schemas/PaymentRequest'
     ManualLedgerCheck:
+      description: Manual ledger check needed message
```
Comment (Kirsten1822) [inline docs/asyncapi.yaml:46]: Is this description correct? I'm not sure I really understand it

File: docs/asyncapi.yaml [hunk 2]
```diff
@@ -31,356 +31,444 @@ channels:
...
-                type: string
-                description: System identifier request originates from
-              value:
-                type: number
-                description: Integer net value of request, ie sum of all invoice lines
+            $ref: '#/components/schemas/PaymentRequest'
           completedPaymentRequests:
             type: array
-            description: Payment request after both delta calculations and automated ledger assignment
+            description: >-
```
[the remark this answers is not in the document]
Comment (johnwatson484) [reply docs/asyncapi.yaml:55]: Allows you to have multi-line descriptions so you don't need a really long single line

[the remark this answers is not in the document]
Comment (Kirsten1822) [reply docs/asyncapi.yaml:55]: Didn't know that - wonder if that's an idea for the AsyncAPI Confluence page?

File: docs/asyncapi.yaml (hunk 1 above)
Comment (Kirsten1822) [reply docs/asyncapi.yaml]: Yeah, I'd say so

File: docs/asyncapi.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (johnwatson484) [reply docs/asyncapi.yaml:55]: Yeah good idea :)

**Review by Kirsten1822 — APPROVED** [review]
Happy to approve :)

**Pull request merged**